layouts/hub-page.tsx

```diff
@@ -25,7 +27,7 @@ const HubPageLayout = ({children}: {children: React.ReactNode}) => {
       <div className="page-container flex min-h-[calc(100vh-(54px+95px))] flex-col items-center">
         <div className="info-container min-w-full min-h-[100px]">
           <Header>
-            <InsightHeader insight={insight} repositories={repositories} />
+            <InsightHeader insight={insight} repositories={repositories} insightId={insightId} isOwner={user?.user_metadata.sub === insight?.user_id} />
```

Comment:
if we need the user id `user?.user_metadata.sub` in more places in the app, it could be a good idea to return it from the hook, right next to `user`

Reply:
Wouldn't that be better done in another issue, in which we would see where user ids are needed in the app and change it?

Reply:
no, because it's not used anywhere else, that is also why I am suggesting doing it here; your PR would benefit most from it and it would help future contributors.